pkg/common/scheduler/statefulset/scheduler.go

```diff
@@ -47,8 +47,10 @@ type SchedulerPolicyType string
 const (
 	// MAXFILLUP policy type adds replicas to existing pods to fill them up before adding to new pods
 	MAXFILLUP SchedulerPolicyType = "MAXFILLUP"
-	// EVENSPREAD policy type spreads replicas uniformly across failure-domains such as regions, zones, nodes, etc
+	// EVENSPREAD policy type spreads replicas uniformly across zones to reduce impact of failure
 	EVENSPREAD = "EVENSPREAD"
```

Comment:
Should we differentiate this by adding ZONE to the policy name?

Reply:
Irrelevant since recent convo with Lionel about one HA scheduler type with all of above.